labgrid/driver/power/eth008.py

```diff
@@ -0,0 +1,42 @@
+"""
+This driver implements a power port for the robot electronics 8 relay
+outputs board.
+
+Driver has been tested with:
+* ETH008 - 8 relay outputs
+"""
+
+import requests
+from ..exception import ExecutionError
+
+PORT = 80
+
+def power_set(host, port, index, value):
+    index = int(index)
+    assert 1 <= index <= 8
+    # access the web interface...
+    value_str = "A" if value else "I"
+    response = requests.get(
+        f"http://{host}:{port}/io.cgi?DO{value_str}{index}"
+    )
+    response.raise_for_status()
+    
+    # Check, that the port is in the desired state
+    state = get_state(response, index)
+    if state != value:
+        raise ExecutionError(f"failed to set port {index} to status {value}")
+
+def power_get(host, port, index):
+    index = int(index)
+    assert 1 <= index <= 8
+    # get the contents of the main page
+    response = requests.get(f"http://{host}:{port}/io.cgi?relay")
+    
+    response.raise_for_status()
+    state = get_state(response, index)
+    return state
+
+def get_state(request, index):
+    value = request.text.split()[1][index-1]
+    states = {"0": False, "1": True}
+    return states[value]
```

Comment:
```suggestion
    bool(int(value))
```
Maybe that's sufficient?

Reply:
Sorry what is wrong with my pullrequest? 
The Codecov Report fails, because of Indirect changes in labgrid/remote/coordinator.py.

What is the next step to get the pullrequest merged?

Reply:
Don't worry about codecov, it's not a required check for this PR to get merged. Next step for this PR is to maybe apply suggestion from @Bastian-Krause if it works.

Reply:
Changes applied.